foundryconfig.example.json

```diff
@@ -1,4 +1,4 @@
 {
   "dataPath": "PATH_TO_DATA_FOLDER/FoundryVTT",
   "foundryUri": "http://localhost:30001"
-}
+}
```

Comment:
No idea what the change was here

Reply:
They killed the terminal line break. Better add it back!